spec/requests/admin/users_manage_spec.rb

```diff
@@ -137,6 +137,28 @@ def call_ghost
       expect(user.roles.last.name).to eq("comment_banned")
     end
 
+    it "selects super admin role when user was banned" do
+      user.add_role :banned
+      user.reload
+
+      params = { user: { user_status: "Super Admin", note_for_current_role: "they deserve it for some reason" } }
+      patch "/admin/users/#{user.id}/user_status", params: params
+
+      expect(user.roles.count).to eq(1)
+      expect(user.roles.last.name).to eq("super_admin")
+    end
+
+    it "does not allow non-super-admin to doll out admin" do
+      super_admin.remove_role(:super_admin)
+      super_admin.add_role(:super_admin)
+      super_admin.reload
+
+      params = { user: { user_status: "Super Admin", note_for_current_role: "they deserve it for some reason" } }
+      patch "/admin/users/#{user.id}/user_status", params: params
+
+      expect(user.has_role?(:super_admin)).not_to be false
+    end
```

Comment:
Sorry, what's going on in this test? Are you trying to test that a super admin cannot give a user `super_admin`? Or are you trying to test that a non-super admin cannot give the admin role -- which I think _should_ be allowed, right?